gradle.properties

```diff
@@ -2,4 +2,4 @@ systemProp.currentTestMisha=FirstTest
 systemProp.currentTestSkizim=SecondTest
 
 # chrome || firefox
-systemProp.browser=firefox
\ No newline at end of file
+systemProp.browser=chrome
```

Comment:
@sergeykizim Don't need to push such changes everytime. Those are only your local configuration.